ssg/build_yaml.py

```diff
@@ -1987,3 +1998,89 @@ def export_ocil_to_file(self, filename):
             test_actions.append(action)
             questions.append(boolean_question)
         tree.write(filename)
+
+
+class Platform(XCCDFEntity):
+
+    KEYS = dict(
+        name=lambda: "",
+        original_expression=lambda: "",
+        xml_content=lambda: "",
+        bash_conditional=lambda: "",
+        ansible_conditional=lambda: "",
+        ** XCCDFEntity.KEYS
+    )
+
+    MANDATORY_KEYS = [
+        "name",
+        "xml_content",
+        "original_expression",
+        "bash_conditional",
+        "ansible_conditional"
+    ]
+
+    prefix = "cpe-lang"
+    ns = PREFIX_TO_NS[prefix]
+
+    @classmethod
+    def from_text(cls, expression, env_yaml):
+        test = env_yaml["product_cpes"].algebra.parse(
```

Comment:
I wonder why the `Algebra` instance is a member of the `ProductCPEs` class. Is it an optimization to prevent creating many instances of the `Algebra` class? Otherwise it seems that you could create an instance of `Algebra` here, without having to access the instance of `ProductCPEs` class.